app.js

```diff
@@ -19,6 +19,20 @@ app.use('/static', express.static('public'));
 
 app.get('/', (req, res) => res.render('index'));
 
+app.get('/dashboard/:id/:page?', function(req, res) {
+  var pageParam = req.params.page;
+  var pageName = 'dashboardView';
+  if (pageParam === 'edit') pageName = 'dashboardEdit';
```

Comment:
This can be shortened to
```var pageName = pageParam === 'edit' ? 'dashboardView' : 'dashboardEdit```

Reply:
It's not clear now, but when I create the categories page, I will add `if (pageParam === 'categories') pageName = 'dashboardCategories;` below line 25, so I think it would be better if I keep this format.